diff --git a/es/Image/components/usePreview.mjs b/es/Image/components/usePreview.mjs
index 1bbed767ed6d4454d97eeb0e3fb7f0b899a0205e..e2df32c748b799466282e0f06f27227d983a23e5 100644
--- a/es/Image/components/usePreview.mjs
+++ b/es/Image/components/usePreview.mjs
@@ -43,7 +43,7 @@ const usePreview = (props) => {
 				onOpenChange?.(open);
 				onVisibleChange?.(open, !open);
 			},
-			rootClassName: cx(styles.preview, rootClassName),
+			classNames: { root: cx(styles.preview, rootClassName) },
 			styles: { mask: { backdropFilter: "blur(8px)" } },
 			...rest
 		};
diff --git a/es/Image/components/usePreviewGroup.mjs b/es/Image/components/usePreviewGroup.mjs
index 417bb367ce6a26653c91c7203a472a8b07220525..d3f43ed19ad605da81bad4e10132847de1e914bd 100644
--- a/es/Image/components/usePreviewGroup.mjs
+++ b/es/Image/components/usePreviewGroup.mjs
@@ -43,7 +43,7 @@ const usePreview = (props) => {
 				onOpenChange?.(open, info);
 				onVisibleChange?.(open, !open, info.current);
 			},
-			rootClassName: cx(styles.preview, rootClassName),
+			classNames: { root: cx(styles.preview, rootClassName) },
 			...rest
 		};
 	}, [
diff --git a/es/Modal/Modal.mjs b/es/Modal/Modal.mjs
index 81ec2ee1e229209afd36d549caea2d838fbdd2e4..2f71314ab740561cafe7493fee77c9fedbd35637 100644
--- a/es/Modal/Modal.mjs
+++ b/es/Modal/Modal.mjs
@@ -67,7 +67,7 @@ const Modal$1 = memo(({ panelRef, allowFullscreen, children, title = " ", classN
 		theme: { token: { colorBgElevated: cssVar.colorBgContainer } },
 		children: /* @__PURE__ */ jsx(Modal, {
 			closable: true,
-			maskClosable: true,
+			mask: { closable: true },
 			cancelText,
 			className: cx(styles.content, className),
 			closeIcon: /* @__PURE__ */ jsx(Icon_default, {
